src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEParameterSymbol.cs

```diff
@@ -980,12 +1016,12 @@ public override ImmutableArray<CSharpAttributeData> GetAttributes()
                 }
 
                 bool filterIsReadOnlyAttribute = this.RefKind == RefKind.In;
+                bool filterLifetimeAnnotationAttribute = Scope != DeclarationScope.Unscoped;
```

Comment:
>Scope != DeclarationScope.Unscoped [](http://example.com/codeflow?start=57&length=34)

I think the attribute should be filtered out even in this condition. It looks like it can be used to encode this value as well.